src/common/constants.js

```diff
@@ -1,7 +1,9 @@
 // @flow
-const ITEM_TYPE_FILE: 'file' = 'file';
-const ITEM_TYPE_FOLDER: 'folder' = 'folder';
-const ITEM_TYPE_WEBLINK: 'web_link' = 'web_link';
+import { TYPE_FILE, TYPE_FOLDER, TYPE_WEBLINK } from '../constants';
+
+const ITEM_TYPE_FILE = TYPE_FILE;
```

Comment:
Can we always directly import from src/constants or what is the reason for re-exporting the constants here?

Reply:
I don't think there is an intentional reason why there are redundant constants here, the only difference is the variable names. I can refactor them in this PR

Reply:
I think @greg-in-a-box and I looked into this once and it was related to types? might want to double check if we need to keep it the way it was

Reply:
https://stackoverflow.com/a/64145916/18705593

Reply:
I cant remember what was going on here, I think there was a type issue but that was because we were trying to convert to TS probably

Reply:
I reverted this specific file just to be safe. I looked at some of the git blames and it looks like it might cause flow-related issues in consuming libraries